test/queries.test.tsx

```diff
@@ -43,8 +40,6 @@ describe('declareQueries', () => {
     const mounted = mount(<WithFoo token="foo" />);
     expect(mounted).toMatchSnapshot();
     expect(render.mock.calls.length).toBe(3);
-    expect(render.mock.calls[2][0].foo.ready).toBe(true);
-    expect(render.mock.calls[2][0].foo.loading).toBe(false);
     expect(render.mock.calls[2][0].foo.value).toBe('foo');
```

Comment:
I was very undecided if I should test more but then I figured it could be a different PR